src/main/java/frc/robot/Robot.java

```diff
@@ -158,27 +158,17 @@ public void testPeriodic() {
    */
   public void driveSpeed() {
     if (!isInControlPanelMode) {
+      // Left thumb stick
       double rawAxis1 = driveController.getRawAxis(1);
+      // Right thumb stick
       double rawAxis4 = driveController.getRawAxis(4);
-
-      // Enabling slow speed toggle on the joystick for drive speed
-      if (driveController.getRawAxis(2) > 0.5) {
-        slowToggle = true;
-      } else {
-        slowToggle = false;
-      }
-
-      // Enabling high speed toggle on the joystick for drive speed
-      if (driveController.getRawAxis(3) > 0.5) {
-        highToggle = true;
-      } else {
-        highToggle = false;
-      }
+      double rawAxis2 = driveController.getRawAxis(2);
+      double rawAxis3 = driveController.getRawAxis(3);
```

Comment:
These two axes should be documented, like the above two.

Reply:
Yes, I was going to document them upon confirmation of what they actually are on the joystick.